cmd/vclusterctl/cmd/platform/connect/space.go

```diff
@@ -31,26 +31,26 @@ type SpaceCmd struct {
 	log log.Logger
 }
 
-// newSpaceCmd creates a new command
-func newSpaceCmd(globalFlags *flags.GlobalFlags, defaults *pdefaults.Defaults) *cobra.Command {
-	cmd := &SpaceCmd{
+// newNamespaceCmd creates a new command
+func newNamespaceCmd(globalFlags *flags.GlobalFlags, defaults *pdefaults.Defaults) *cobra.Command {
+	cmd := &NamespaceCmd{
 		GlobalFlags: globalFlags,
 		log:         log.GetInstance(),
 	}
 
-	description := product.ReplaceWithHeader("connect space", `
-Creates a new kube context for the given space.
+	description := product.ReplaceWithHeader("connect namespace", `
+Creates a new kube context for the given vCluster platform namespace.
 
 Example:
-vcluster platform connect space
-vcluster platform connect space myspace
-vcluster platform connect space myspace --project myproject
+vcluster platform connect namespace
+vcluster platform connect namespace myspace
+vcluster platform connect namespace myspace --project myproject
 ########################################################
 	`)
 	useLine, validator := util.NamedPositionalArgsValidator(false, false, "SPACE_NAME")
 	c := &cobra.Command{
-		Use:   "space" + useLine,
-		Short: "Creates a kube context for the given space",
+		Use:   "namespace" + useLine,
+		Short: "Creates a kube context for the given vCluster platform namespace",
```

Comment:
Wasn't exactly sure if we should use plain `namespace` or a more "branded" `vCluster platform namespace`...